lib/bot-service.js

```diff
@@ -4,13 +4,13 @@ const Store = require('./store');
 const https = require('https');
 const jwt = require('jsonwebtoken');
 
-const HOST = 'api.bankingofthings.io';
+const HOST = 'iot.bankingofthings.io';
 const PORT = 443;
 const ENDPOINT_PREFIX = '/bot_iot/';
```

Comment:
doesn't this change too? should we?

Reply:
Yes it could, and perhaps we should.
Ill put it on my backlog and fix it in the coming days.
We also need to make sure the other SDK's make the same change @Lokesh-K-Haralakatta  could you align this?

For now ill close this PR.